src/libs/config/src/lib.rs

```diff
@@ -54,13 +54,21 @@ pub mod system {
 
 pub mod syscomm {
 
+    ///
+    /// # Description
+    ///
+    /// Provides the maximum length of a message we receive. We set a limit to
+    /// prevent a malformed payload to trigger unbounded allocations in linuxd.
+    ///
+    pub const MAX_MESSAGE_LEN: usize = 128 * 1024 * 1024;
+
```

Comment:
Why is this new constant required?

Reply:
Its not exactly a LINUXD message, so that is why I added a new constant. I did not want to refactor MAX_LINUXD in this PR to not add more noise but, if you are happy about it, I can refactor it in this PR.

Reply:
Done!